documentation/blog/2025-04-01-top-5-mcp-servers/index.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Top 5 MCP Servers I Use as a Developer with Goose Extensions"
+description: "These 5 MCP servers help me automate my workflow and make me a better developer."
+date: 2025-03-31
+authors: 
+    - adewale
+---
+
+![blog cover](mcp-servers-cover.png)
+
+As a developer, finding the right tools that seamlessly work together can feel like discovering a superpower. And when you have a working process, it can sometimes be difficult to try out new tools.
+
+With the introduction of MCPs, AI agents like Goose is able to plug in to my existing tools and the only thing that changes with my workflow, is the much welcomed automation that comes with it. I still do the same things I do, but backed by AI, I can now do them faster and with more confidence.
+
+Today, I'm excited to share not just my favorite MCP servers, but the ones I actually use almost everyday with real applications that you can probably relate to as well.
+
+<!--truncate-->
+
+:::tip
+You can ask Goose `"What can I do with xxxx Extension?"` to get a list of all the features and example use cases you can try out. 
+:::
+
+## GitHub MCP Server: Everything GitHub
+
+The [GitHub MCP Server](/docs/tutorials/github-mcp) comes with quite a lot of functionality. Can help you create issues, pull requests, repositories, branches etc but what I use it for the most is in reviewing and understanding pull requests.
```

Comment:
```suggestion
The [GitHub MCP Server](/docs/tutorials/github-mcp) comes with quite a lot of functionality. It can help you create issues, pull requests, repositories, branches etc but what I use it for the most is in reviewing and understanding pull requests.
```